components/tyk-mdcb/values.yaml

```diff
@@ -303,7 +307,17 @@ mdcb:
     externalTrafficPolicy: Local
     # annotations corresponds to the annotations which will be added into Tyk Dashboard Service.
     annotations: {}
-
+  
+  http_service:
```

Comment:
You also need to create service.

Reply:
@sredxny You can skip creation of service as http port is already exposed in a service.
Please remove `http_service` section from values file and update PR accordingly.

Reply:
thank you